webpack.config.js

```diff
@@ -0,0 +1,25 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  cache: true,
+  entry: "./modules/index.js",
+  output: {
+    path: path.join(__dirname, '/dist'),
+    filename: 'Radium.js',
```

Comment:
We've been using uppercase everywhere, but yeah, it probably should be lowercase.